main.c

```diff
@@ -238,6 +247,19 @@ enum option_indices {
         *t++ = 0;\
     } while (false)
 #endif
+
+struct preset {const char* name; const char* settings; int exp; int money;};
+const static struct preset presets[] = {
+    {"First-Timer", "rel", 300, 300},
+    {"Beginner", "rel", 200, 250},
+    {"Advanced", "rlABGSCd67", 150, 200},
```

Comment:
spoiler log `on` by default for advanced/pro/hell/hell+? Also not sure hell+ is a good idea :D

Reply:
tbh I don't really see any reason to not generate a log file unless it's specifically a race but I can also remove it for those. your call.
hell+ is for the Menblocks of this world. also is just wanted a 1% preset :D

Reply:
maybe let's call it menblock for the memes?